services/course.js

```diff
@@ -71,6 +71,10 @@ const courseService = {
 
     await course.validate()
     await course.save()
+  },
+
+  getCourseById: async (id) => {
+    return await Course.findById(id).lean().exec()

```

Comment:
The same here. Move above after getCourses

Reply:
> The same here. Move above after getCourses

moved that